### PR TITLE
Change InsideModal according to recently modification on Modal

### PR DESCRIPTION
**Pull request by futurechallenger**
1. Remove `CloseButton` in `InsideModal`, as it's not needed in `Modal`.
2. Add `showCloseBtn` and `onCancel` props according to `Modal` prop types.
3. Add `showCloseBtn` prop in `InsideModal`, it's default value is true.

So the modal can be closed.

### CODE REVIEW

File: packages/ringcentral-widgets/components/InsideModal/index.js [hunk 1]
```diff
@@ -5,37 +5,17 @@ import Modal from '../Modal';
 import styles from './styles.scss';
 import InsideModalClose from '../../assets/images/InsideModalClose.svg';
```
Comment (Hazyzh) [inline packages/ringcentral-widgets/components/InsideModal/index.js]: Seems like can remove this line at the same time.

Comment (futurechallenger) [reply packages/ringcentral-widgets/components/InsideModal/index.js]: Done!

File: packages/ringcentral-widgets/components/InsideModal/index.js [hunk 2]
```diff
@@ -44,8 +24,10 @@ export default function InsideModal(
       maskClassName={classnames(styles.mask, maskStyle)}
       modalClassName={classnames(styles.modal, modalStyles)}
       contentClassName={classnames(styles.content, contentStyle)}
-      closeBtn={closeBtn}
-      show={show}>
+      show={show}
+      showCloseBtn={showCloseBtn}
+      onCancel={onClose}
+    >
```
Comment (huanhulan) [inline packages/ringcentral-widgets/components/InsideModal/index.js]: how about add a`clickOutToClose` props?

Comment (futurechallenger) [reply packages/ringcentral-widgets/components/InsideModal/index.js]: It's cool! Done!

**Pull request merged**